README.md

```diff
@@ -42,64 +58,75 @@ Stay updated with the latest developments on the
 
 ## Installation
 
-1. Clone or download the desired version of PECOS.
-2. Navigate to the root directory, where `pyproject.toml` is located.
-3. Install using pip:
+### Python Package
+
+To install the main Python package:
 
 ```sh
-pip install .
+pip install quantum-pecos
 ```
 
-To install optional dependencies, such as for Wasm support or state vector simulations, see `pyproject.toml` for list of
-options. To install all optional dependencies use:
+This will install both `quantum-pecos` and its dependency `pecos-rslib`.
+
+For optional dependencies:
 
 ```sh
-pip install .[all]
+pip install quantum-pecos[all]
 ```
 
-Certain simulators have special requirements and are not installed by the command above. Installation instructions for these are provided [here](#simulators-with-special-requirements).
+**NOTE:** The `quantum-pecos` package is imported like: `import pecos` and not `import quantum_pecos`.
 
-For development, use (while including installation options as necessary):
+Certain simulators have special requirements and are not installed by the command above. Installation instructions for
+these are provided [here](#simulators-with-special-requirements).
 
-On Linux/Mac:
 
-```sh
-python -m venv .venv
-source .venv/bin/activate
-pip install -U pip setuptools
-pip install -r requirements.txt
-make metadeps
-pre-commit install
-pip install -e .
-```
+### Rust Crates
 
-On Windows:
+To use PECOS in your Rust project, add the following to your `Cargo.toml`:
 
-```sh
-python -m venv .venv
-.\venv\Scripts\activate
-pip install -U pip setuptools
-pip install -r requirements.txt
-make metadeps
-pre-commit install
-pip install -e .
+```toml
+[dependencies]
+pecos = "0.x.x"  # Replace with the latest version
 ```
 
-See `Makefile` for other useful commands.
+## Development Setup
 
-Tests can be run using:
+For developers who want to contribute or modify PECOS:
 
-```sh
-pytest tests
-```
+1. Clone the repository:
+   ```sh
+   git clone https://github.com/PECOS-packages/PECOS.git
+   cd PECOS
+   ```
 
-### Simulators with special requirements
+2. Set up the development environment using the makefile (for Windows you may need to use a shell that has access to Linux commands such as utilizing [git bash](https://gitforwindows.org/)):
+   ```sh
+   make venv
+   make build
```

Comment:
Fuller dump:

```bash
📦 Including license file "/home/peter/Downloads/PECOS-feat-rust/python/pecos-rslib/LICENSE"
🍹 Building a mixed python/rust project
🔗 Found pyo3 bindings
🐍 Found CPython 3.10 at /home/peter/Downloads/rust-pecos/bin/python
📡 Using build options features from pyproject.toml
    Finished `dev` profile [unoptimized + debuginfo] target(s) in 0.07s
📦 Built wheel for CPython 3.10 to /tmp/.tmpzQTl39/pecos_rslib-0.6.0.dev6-cp310-cp310-linux_x86_64.whl
✏️  Setting installed package as editable
🛠 Installed pecos-rslib-0.6.0.dev6
Looking in indexes: https://john.campora%40quantinuum.com:****@quantinuumhw.jfrog.io/artifactory/api/pypi/qtm-pypi-vc/simple
Obtaining file:///home/peter/Downloads/PECOS-feat-rust/python/quantum-pecos
  Installing build dependencies ... done
  Checking if build backend supports build_editable ... done
  Getting requirements to build editable ... done
  Preparing editable metadata (pyproject.toml) ... done
INFO: pip is looking at multiple versions of quantum-pecos to determine which version is compatible with other requirements. This could take a while.
ERROR: Could not find a version that satisfies the requirement pecos-rslib==0.6.0.dev6 (from quantum-pecos) (from versions: none)
ERROR: No matching distribution found for pecos-rslib==0.6.0.dev6
make: *** [Makefile:75: build] Error 1

```